rusoto/core/src/client.rs

```diff
@@ -135,6 +135,7 @@ where
     }
 }
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
Should we address this? I put this in so `clippy` would look at the S3 crate - it was hung up on `rusoto_core` and wouldn't look at anything else until this was addressed.

Reply:
This is probably a bigger change -- I'd leave it as-is for now, and we can revisit in a separate PR.